lib/deps/buffer-browser.js

```diff
@@ -0,0 +1,2 @@
+// hey guess what, we don't need this in the browser
+module.exports = {};
```

Comment:
``module.exports`` is already an empty object by default, making this line unnecessary, right?